tests/test_0692_fsspec_reading.py

```diff
@@ -53,13 +54,17 @@ def test_open_fsspec_local():
 @pytest.mark.parametrize(
     "handler",
     [
-        # uproot.source.fsspec.FSSpecSource,
+        uproot.source.fsspec.FSSpecSource,
         uproot.source.s3.S3Source,
         None,
     ],
 )
 def test_open_fsspec_s3(handler):
     pytest.importorskip("s3fs")
+    if sys.version_info < (3, 11):
+        pytest.skip(
+            "https://github.com/scikit-hep/uproot5/pull/1012",
+        )
```

Comment:
@lobis why specifically versions less than 3.11?

Reply:
No particular reason, it just failed on older versions, I never could find out why.